nss-day-cohort-41/react-nutshell-the-hazelnuts
# Clean-up

# Description
- Removed all console logs
- Removed all errors where functions wouldn't break
- Various small syntax fixes.
- Tested all CRUD components for each subpage

## Type of change


# Testing Instructions
- Use all available CRUD components throughout all subpages of app. Ensure no broken links.
- Look over each page for any errors in spelling, grammar, styling, etc.

# Checklist:
